README.md

```diff
@@ -19,6 +19,13 @@ Then install SDL2 and ELF libraries:
 > brew install sdl2 libelf
 ```
 
+If avr-gcc cross-compiler is not installed on your system:
```

Comment:
Are you sure this is necessary? I recall avr-gcc not being required to build sim-arduboy (and libsimavr).

Reply:
At least project generated for Xcode can't build without avr-gcc from brew. Probably some headers from avr-gcc is required.

Reply:
What I think is happening is for some reason the Xcode project is building the entire simavr project instead of just building libsimavr

Reply:
Probably because I use new CMakeFiles.txt for tests:
https://github.com/dlancer/sim-arduboy/blob/linux-cmake/cmake/CMakeLists.txt

Reply:
I tested your original CMakeLists.txt with project generation for Xcode (cmake -G Xcode) and see the same notice about missed avr-gcc when I try build project from Xcode.
MacOS High Sierra 10.13.6 / XCode 10.0 / cmake version 3.12.2

Reply:
Yes, it happens only with CMake. BTW I added CMake support reluctantly. I understand it's good to generate project files for IDEs and could be used to integrate the simulator in a development environment but I consider makefiles to be the primary way to build this project.

Back to your contribution I'm going to merge it and add a note to the readme saying avr-gcc is only needed when building via CMake.